tests/integration/nm/ipv4_test.py

```diff
@@ -89,10 +89,9 @@ def _modify_interface(ipv4_state):
     new_conn = nm.connection.ConnectionProfile()
     new_conn.create(settings)
     conn.update(new_conn)
-    conn.commit(save_to_disk=False)
 
     nmdev = nm.device.get_device_by_name(TEST_IFACE)
-    nm.device.reapply(nmdev, conn.profile)
+    nm.device.modify(nmdev, new_conn.profile)
```

Comment:
Not a related change.
BTW, the usage of `reapply` explicitly here was intentional.. it forces a "reapply", otherwise it fails (unlike our abstraction of "modify" where a fallback is called to activate).

Reply:
This function is only used by test code.
I have it removed from current code base. If you think test code still need it, please add it back
to the test code only.

Reply:
The unused code has been removed. The only use of that function is nm test code. If required please add it back to the test code only.